website/docs/tutorials/image_feature_extraction.md

```diff
@@ -0,0 +1,67 @@
+---
+id: image_feature_extraction
+title: Image Feature Extraction
+sidebar_label: Image Feature Extraction
+---
+
+In this tutorial, we will go through the step-by-step process of generating image features with FasterRCNN feature extractors. MMF provides utility scripts for running image feature extractions using different models (Faster RCNN with X-101 backbones and X-152 backbones). For example: `tools/scripts/features/extract_features_vmb.py`. The script allows you to optionally parallelize the feature extraction execution. 
+
+Here are the steps in a nut shell:
+- Prerequisites: Install vqa-maskrcnn-benchmark
+- Download the dataset (not covered in this tutorial)
+- Identify which vision feature extractor you'd like to use
+- Extract image features 
+- Extract image features with slurm
+
+## Prerequisites and Intallation
+
+Follow the prerequisites for installation of the vqa_maskrcnn-benchmark [here](https://gitlab.com/vedanuj/vqa-maskrcnn-benchmark#installation) 
```

Comment:
Can we move the prerequisites here so that they are under our control. Then, we can update that repo's README to point here instead.